README.md

```diff
@@ -222,6 +222,10 @@ Emitted when the request is aborted. This event will be followed shortly
 by an `error` event. In practice you do not need to handle this event.
 
 #### 'progress' (bytesReceived, bytesExpected)
+Emitted when bytes have been received for the form.
+
+`bytesReceived` - the amount of bytes received for this form so far.
+`bytesExpected` - this value is either an integer, 0, or null.  If the `Content-Length` request header is set, it will be set to that.  If the `Transfer-Encoding` request header is set, it will be set to 0 and by default it will be set to null.
```

Comment:
So 0 is an integer, so I'm not sure if that should be listed out specifically here. The second sentence doesn't sem to match with the code in this module, particularly that if Transfer-Encoding is set, it sets it to `null`, not to zero.

Reply:
🤦‍♂ doh!  will update